.tugboat/config.yml

```diff
@@ -67,7 +67,7 @@ services:
         - BUILD_TYPE=vagovprod node ./scripts/yarn/vets-website-assets.js
 
         # Build the static pages. Set for self-signed certs
-        - NODE_EXTRA_CA_CERTS=/etc/ssl/certs/ca-certificates.crt BUILD_OPTION=static yarn export
+        - NODE_EXTRA_CA_CERTS=/etc/ssl/certs/ca-certificates.crt BUILD_OPTION=static yarn export --DEBUG
```

Comment:
@timcosgrove Should we just leave this on inside of tugboat? Feels like it should be fine, can't think of a reason it would impact us negatively

Reply:
I would take it out here. It's not part of this PR.